spec/models/manageiq/providers/redhat/infra_provider_spec.rb

```diff
@@ -18,4 +18,22 @@
     h = FactoryGirl.create(:ems_redhat, :hostname => "h")
     h.rhevm_metrics_connect_options(:hostname => "i")[:host].should == "i"
   end
+
+  it "connect Metrics" do
+    h = FactoryGirl.create(:ems_redhat, :hostname => "h")
+    expect(h).to receive(:metrics_connect)
+    h.connect(:service => "Metrics")
```

Comment:
@matthewd Thank you and I agree this is a better solution. It is similar to what I had originally tried but since that surfaces another issue, which is we comment out ovirt_metrics from our Gemfile [1] causing the pre-require of the library to fail, I had gone with the weaker test. Please pass along any suggestions on how to address [1].

Thanks again! JoeV

[1] https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L73